packages/main/src/DateComponentBase.ts

```diff
@@ -92,8 +96,18 @@ class DateComponentBase extends UI5Element {
 
 	static i18nBundle?: I18nBundle;
 
+	/**
+	 * Cached instance of DateFormat with a format pattern of "YYYY-MM-dd".
+	 * Used by the getISOFormat method to avoid creating a new DateFormat instance on each call.
+	 *
+	 * @type {DateFormat}
+	 * @private
+	 */
+	_isoFormatInstance: DateFormat | null;
```

Comment:
This is enough:
```
_isoFormatInstance?: DateFormat;
```

It's enough because we never reset the value to null and it's okey to be undefined or defined when it's going to be used.